templates/students/update_profile.html

```diff
@@ -21,27 +21,14 @@ <h2>
             Update Student Profile
         </h2>
     </div>
-    <form method="POST" class="needs-validation" enctype="multipart/form-data" novalidate>
+    <form method="POST" class="needs-validation" id="student-profile-update-form" enctype="multipart/form-data" novalidate>
         <div class="row justify-content-center mb-5">
             <div class="col-lg-9 col-xl-8 border rounded shadow px-5 py-4">
                 <div class="row justify-content-center">
                     {% csrf_token %}
                     <div class="col-md-4 mb-4 align-self-center text-center">
-                        {% if profile.profile_image %}
-                        <img class="rounded-circle" style="height: 150px; width: 150px;"
-                            src="{{ profile.profile_image.url }}" alt="profile_image">
-                        {% elif profile.gender == 'M' %}
-                        <img class="rounded-circle" style="height: 150px; width: 150px;"
-                            src="{% static 'home/images/man.png' %}" alt="profile_image">
-                        {% elif profile.gender == 'F' %}
-                        <img class="rounded-circle" style="height: 150px; width: 150px;"
-                            src="{% static 'home/images/woman.png' %}" alt="profile_image">
-                        {% else %}
-                        <img class="rounded-circle" style="height: 150px; width: 150px;"
-                            src="{% static 'home/images/man.png' %}" alt="profile_image">
-                        {% endif %}
+                        {% include 'globals/updatable_profile_image.html' %}
```

Comment:
Don't send the complete context here but only the `profile_image_url`. Use `with` and `only` attributes for this (see documentation).